CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# dbt_salesforce v0.6.0
+## Features
+- Updating package structure to align with most recent standards 
```

Comment:
Can you add more context here. Similar to what you did within the source pacakge.